### PR TITLE
Access Control for Breaking News

### PR DESCRIPTION
**Pull request by robertberry-zz**
CC @piuccio There is an endpoint at /acl now that returns

``` json
{
    "breakingNews": true
}
```

if the user has access. (`false` there if not)

If a user attempts to edit or delete a collection that is on `breaking-news` and they don't have access to, they'll be given a 401 Not Authorized error.

### CODE REVIEW

**Comment by robertberry** [comment]
Oh and to give someone access log into the AWS panel and add their email to BreakingNewsAccessControlList in DynamoDB.

File: facia-tool/conf/routes [hunk 1]
```diff
@@ -70,3 +70,4 @@ POST          /stories-visible/*containerType   controllers.StoriesVisibleContro
 
 #Vanity URL
 GET           /breaking-news                    controllers.VanityRedirects.breakingnews
+GET           /acl                              controllers.AccessControlListController.test()
```
Comment (piuccio) [inline facia-tool/conf/routes]: we need the same route in `facia-end-to-end`

Comment (robertberry) [inline facia-tool/conf/routes]: ok done

**Comment by gklopper** [comment]
Talk Monday about this... ACL is a PANDA feature and we need to move this to PANDA...

**Comment by robertberry** [comment]
Yeah I know. This is a temporary thing - editorial don't want anyone sending breaking news alerts that shouldn't be.

**Comment by gklopper** [comment]
>This is a temporary thing

Heard that one before :rage: - maybe a switch with a sensible date...

**Comment by robertberry** [comment]
Lol. Well isn't this supposedly all moving over to the tool's team?

**Comment by piuccio** [comment]
Closing this, apparently it's not high priority and it can wait for the permission service

**Pull request closed without merging**